lib/defaults/defaults.go

```diff
@@ -762,23 +751,33 @@ var (
 
 	// FIPSCiphers is a list of supported FIPS compliant SSH ciphers.
 	FIPSCiphers = []string{
-		"aes128-ctr",
-		"aes192-ctr",
+		"aes256-gcm@openssh.com",
 		"aes256-ctr",
+		"aes192-ctr",
 		"aes128-gcm@openssh.com",
+		"aes128-ctr",
 	}
 
 	// FIPSKEXAlgorithms is a list of supported FIPS compliant SSH kex algorithms.
 	FIPSKEXAlgorithms = []string{
 		"ecdh-sha2-nistp256",
 		"ecdh-sha2-nistp384",
-		"echd-sha2-nistp521",
 	}
 
 	// FIPSMACAlgorithms is a list of supported FIPS compliant SSH mac algorithms.
 	FIPSMACAlgorithms = []string{
 		"hmac-sha2-256-etm@openssh.com",
+		"hmac-sha2-512-etm@openssh.com",
 		"hmac-sha2-256",
+		"hmac-sha2-512",
+	}
+
+	// FIPSPubKeyAuthAlgorithms is a list of supported FIPS compliant SSH public key authentication algorithms.
+	FIPSPubKeyAuthAlgorithms = []string{
```

Comment:
This is new in golang/go#61244 / https://github.com/golang/crypto/commit/eb61739cd99fb244c7cd188d3c5bae54824e781d.